health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -70,26 +70,27 @@ export const UICustomizations = {
-            // const [localIsActive, setLocalIsActive] = useState(rowDataCache[serviceCode].isActive);
+            const [localIsActive, setLocalIsActive] = useState(row?.ServiceRequest?.[0]?.isActive);
             const toggle = async () => {
               const prev = row?.ServiceRequest?.[0]?.isActive;
               const sdcode = row?.ServiceRequest?.[0]?.code;
               const res = await updateServiceDefinition(tenantId, !prev, sdcode);
+              setLocalIsActive(!localIsActive);
               if(res)
               {
 
               }
             };
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)